lib/suite.js

```diff
@@ -493,6 +493,15 @@ Suite.prototype.appendOnlySuite = function(suite) {
   this._onlySuites.push(suite);
 };
 
+/**
+ * Marks a suite to be `only`.
+ *
+ * @private
+ */
+Suite.prototype.markOnly = function() {
+  this.parent.appendOnlySuite(this);
```

Comment:
I note that `parent` can be `null`.  I'm not sure how someone could cause this to happen, but might as well do a `this.parent && this.parent.appendOnlySuite(this)`

Reply:
Done in #4282